app/views/links/index.html.erb

```diff
@@ -5,14 +5,16 @@
     <tr>
       <th>Original</th>
       <th>Srink URL</th>
+      <th>Visits</th>
       <th>Actions</th>
     </tr>
   </thead>
   <tbody>
     <% @links.each do |link| %>
       <tr id="<%= "link-#{link.id}" %>">
         <td><%= link.url %></td>
-        <td><a href="<%= redirect_url(link.token) %>"><%= redirect_url(link.token) %></a></td>
+        <td><a href="<%= redirect_url(link.token) %>" data-turbolinks="false"><%= redirect_url(link.token) %></a></td>
```

Comment:
I'm not entirely sure why, but turbolinks seems to be generating two requests when the user interacts with these links - an XHR visit and a full page navigation event. This results in two visit events being captured when the link is pressed in the user's links list. Here we tell turbolinks to NOT generate the XHR request and simply let the browser follow the location redirect from the srink server.